main.py

```diff
@@ -1,6 +1,6 @@
 from game import Game
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
`black` started a holy war about which was better `'` or `"`. Black chooses `"`. It's really unfortunate. Cause I hate it. Lol. I always use [skip-string-normilzation](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings). Long story short, be consistent, and it doesn't really matter.

Reply:
L O L